src/components/Main/Schedule/NewScheduleForm.tsx

```diff
@@ -29,59 +26,49 @@ interface NewScheduleFormProps {
 }
 
 export const NewScheduleForm = ({ onSubmit, onClose }: NewScheduleFormProps) => {
-  const today = new Date();
-  const tomorrow = new Date(today.setDate(today.getDate() + 1));
-  const [scheduleTitle, setScheduleTitle] = useState<TextFieldState>(
-    createTextFieldState(''),
+  const tomorrow = new Date(new Date().setDate(new Date().getDate() + 1));
+  const classes = useStyles();
+
+  const [title, setTitle, setTitleError, resetTitleError] = useValidatedField(
+    '',
+    'Title must not be blank and be under 32 characters long',
```

Comment:
Shorter and less ambiguous: 
`'Title must be filled and less than 32 characters',`

or:
`'Must have a title that is less than 32 characters',` 
and user assumes "must have a title" means > 0 chars.